muxdb/cache.go

```diff
@@ -51,13 +51,8 @@ func (c *cache) log() {
 	last := c.lastLogTime.Swap(now)
 
 	if now-last > int64(time.Second*20) {
-		log1, ok1 := c.nodeStats.ShouldLog("node cache stats")
-		log2, ok2 := c.rootStats.ShouldLog("root cache stats")
-
-		if ok1 || ok2 {
-			log1()
-			log2()
-		}
+		c.nodeStats.Log("node cache stats")
```

Comment:
The overarching method is also called `log`. Even though it applies to `cache` instead, should we rename any of them?

I also think that the former `ShouldLog` applied just within the package, should it be lower-case (not sure if we are being strict about this though, just my first review, feel free to dismiss this :p )